src/style/navbar.css

```diff
@@ -9,14 +9,14 @@
   border: none;
   text-decoration: none;
   font-size: 20px;
-  color: rgb(211, 200, 200);
+  color: rgb(202, 198, 198);
   font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
 }
 
 .item:hover {
-  color: rgb(255, 255, 255);
+  color: rgb(113, 113, 113);
 }
 
 .active {
```

Comment:
- [ ] Please make sure to add a border between the navbar menu options to follow the wireframe design.

### Your design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/af0c2dc3-22ae-42a2-a8f0-54f11f831bd6)

### Wireframe design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/c1961243-2ebf-4fbe-906d-ef1c48864302)